main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
-# Starting a Project
+# Your First Agoric Dapp
 
-After you've [installed the Agoric SDK](./README.md) (_recall: use `agoric --version` to confirm._), you're ready for your first _Agoric Dapp_ (decentralized application).
 
+# How to Get Help
```

Comment:
There should be just one 1st level heading on a page. The rest should be 2nd (or 3rd...) level.

```suggestion
## How to Get Help
```

Reply:
done